gamemodes/terrortown/gamemode/shared/sh_lang.lua

```diff
@@ -38,18 +38,24 @@ function LANG.SetupFiles(lang_path, deepsearch)
 	local file_paths = {}
 
 	if deepsearch then
-		local sub_folders = ({file.Find(lang_path .. "*", "LUA")})[2] or {}
+		local _, sub_folders = file.Find(lang_path .. "*", "LUA")
+
+		if not sub_folders then return end
 
 		for k = 1, #sub_folders do
 			local subname = sub_folders[k]
-			local files = ({file.Find(lang_path .. subname .. "/*.lua", "LUA")})[1] or {}
+			local files = file.Find(lang_path .. subname .. "/*.lua", "LUA")
+
+			if not files then return end
```

Comment:
that would break the whole function. Maybe you missed `continue`?

Reply:
Very good spot!